src/components/ebay-tab/template.marko

```diff
@@ -0,0 +1,58 @@
+<div w-bind class=data.classes ${data.htmlAttributes}>
+    <if(data.fake)>
+        <ul class="fake-tabs__items">
+            <for(i, item in data.items)>
+                <var isCurrentItem=parseInt(i) === data.index/>
+                <li
+                    class=[item.classes, 'fake-tabs__item', isCurrentItem ? 'fake-tabs__item--current' : null]
+                    ${item.htmlAttributes}>
+                    <a
+                        aria-current=(isCurrentItem && "page")
+                        href=item.href
+                        w-body=item.renderBody/>
+                </li>
+            </for>
+        </ul>
+        <div class="fake-tabs__content">
+            <var panel=data.panels[0]/>
+            <div class=panel.classes ${panel.htmlAttributes}>
+                <div class="fake-tabs__cell">
+                    <div w-body=panel.renderBody/>
+                </div>
+            </div>
+        </div>
+    </if>
+    <else>
+        <div w-id="items" class="tabs__items" role="tablist">
+            <for(i, item in data.items)>
+                <div
+                    aria-controls="${widget.id}-tabpanel-${i}"
+                    id="${widget.id}-default-tab-${i}"
```

Comment:
Minor - why are they all called "default-tab" (plus id and item number)?  I would see this and think whatever tab I'm looking at is the default.

Reply:
Thanks, I'll update. This was from the Skin example.